core/roboconf-dm/src/main/java/net/roboconf/dm/internal/delegates/InstanceMngrDelegate.java

```diff
@@ -400,6 +400,11 @@ private void deployTarget( ManagedApplication ma, Instance scopedInstance ) thro
 				throw (TargetException) e;
 			else if( e instanceof IOException )
 				throw (IOException) e;
+
+		} finally {
+			synchronized( LOCK ) {
+				scopedInstance.data.remove( Instance.TARGET_ACQUIRED );
```

Comment:
I had not seen the build failed.
That's weird, tests were working on my machine. I will check it.